tools/m2ctx.py

```diff
@@ -42,6 +82,8 @@ def main():
                                      description="Creates a ctx.c file for mips2c. "
                                      "Output will be saved as oot/ctx.c")
     parser.add_argument('filepath', help="path of c file to be processed")
+    parser.add_argument("--fix", "-f", dest="fix", action="store_true", default=False, 
```

Comment:
`--fix` sounds a bit too generic to me. Also, some people may argue this is _breaking_ stuff instead of fixing it. I suggest something like `--custom` instead
```suggestion
    parser.add_argument("--custom", "-c", dest="fix", action="store_true", default=False, 
```